### PR TITLE
Feature - Added onClicked prop to InputWrapper

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Checklist:

- [x] I have updated the CHANGELOG with a summary of my changes
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/atoms/inputWrapper/component.tsx [hunk 1]
```diff
@@ -96,13 +96,20 @@ const StyledMessage = styled.p`
...
+  onClicked?: () => void;
   // isFocussed?: boolean;
 }
 
 export const InputWrapper = (props: IInputWrapperProps): React.ReactElement => {
   // :focus-within is not supported on all browsers so we manually maintain if this particle has a isFocussed child
   const [isFocussed, setIsFocussed] = React.useState(false);
   const theme = useBuiltTheme('inputWrappers', props.variant, props.theme);
+
+  const onClick = () => {
```
Comment (krishan711) [inline src/atoms/inputWrapper/component.tsx:108]:
```suggestion
  const onClicked = () => {
```

File: src/molecules/singleLineInput/component.tsx [hunk 2]
```diff
@@ -77,7 +77,7 @@ export const SingleLineInput = (props: ISingleLineInputProps): React.ReactElemen
     }
   };
 
-  const onClick = (): void => {
+  const onClick = () => {
```
Comment (krishan711) [inline src/molecules/singleLineInput/component.tsx:80]: this should really be onClicked (as should the prop). Can you please update this one and add a todo above the prop saying to change it to `onClicked`

File: src/atoms/inputWrapper/component.tsx (hunk 1 above)
Comment (MoiyadKayda) [reply src/atoms/inputWrapper/component.tsx:108]: it was onClick before

File: src/molecules/singleLineInput/component.tsx (hunk 2 above)
Comment (MoiyadKayda) [reply src/molecules/singleLineInput/component.tsx:80]: but we will have to change this in all other project where we use `singleInputField`

Comment (krishan711) [reply src/molecules/singleLineInput/component.tsx:80]: dont change the prop, just this intenral function. put a TODO above the prop

**Review by krishan711 — APPROVED** [review]

**Pull request merged**